multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt

```diff
@@ -432,6 +432,7 @@ class MadaraGenerator : ThemeSourceGenerator {
         SingleLang("Ultralight Scans", "https://readlight.org", "en"),
         SingleLang("Unemployed Scans", "https://unemployedscans.com", "en", overrideVersionCode = 1),
         SingleLang("Uyuyan Balik", "https://uyuyanbalik.com", "tr", overrideVersionCode = 1),
+        SingleLang("Uzay Manga", "https://uzaymanga.com", "tr", overrideVersionCode = 1),
```

Comment:
Can remove the `overrideVersionCode` since it's the first version.

```suggestion
        SingleLang("Uzay Manga", "https://uzaymanga.com", "tr"),
```